server/sonar-web/src/main/js/apps/projects/filters/IssuesFilter.js

```diff
@@ -31,10 +31,14 @@ export default class IssuesFilter extends React.Component {
     organization: React.PropTypes.object
   }
 
-  renderOption = (option, selected) => {
+  renderOption = (option, selected, value) => {
+    console.log(`testlog`, option, selected, value); // eslint-disable-line
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS2228)